File: fix_status.js
const fs = require('fs');

let content = fs.readFileSync('D:/waterapp/app/inventory/transactions/page.tsx', 'utf-8');

content = content.replace(/row\.status === 'pending'/g, "['pending', '?????', '??? ????????'].includes(row.status)");
content = content.replace(/row\.status === 'approved'/g, "['approved', '?????', '????'].includes(row.status)");

fs.writeFileSync('D:/waterapp/app/inventory/transactions/page.tsx', content, 'utf-8');
console.log('Done');

File: fix_status.py
import sys

with open('D:/waterapp/app/inventory/transactions/page.tsx', 'r', encoding='utf-8') as f:
    content = f.read()

content = content.replace("row.status === 'pending'", "['pending', '?????', '??? ????????'].includes(row.status)")
content = content.replace("row.status === 'approved'", "['approved', '?????', '????'].includes(row.status)")

with open('D:/waterapp/app/inventory/transactions/page.tsx', 'w', encoding='utf-8') as f:
    f.write(content)

print('Done')
